trackhub/track.py

```diff
@@ -703,7 +710,7 @@ def __str__(self):
         s = []
 
         s.append(super(SuperTrack, self).__str__())
-        s.append('superTrack on')
+        s.append('superTrack on' + self.supertrackonoff)
```

Comment:
Andre, is this right? Isn't this going to output something like "superTrack on on" instead of "superTrack on"?

Reply:
Thats the current code on master (see https://github.com/mauranolab/trackhub/blob/225bc567f5c6bbcbf9332ddfb59acdbb4503eb40/trackhub/track.py#L627), and it outputs something like: "superTrack on show" or "superTrack on hide"

Reply:
yes, my bad!